.github/workflows/build_package.yml

```diff
@@ -27,39 +27,39 @@ on:
 
 jobs:
   build_core:
-    name: "${{ matrix.os }} :: Build ${{ matrix.build_package }} Package"
-    runs-on: ${{ matrix.os }}
+    name: "${{ matrix.build-family }} :: Build ${{ matrix.build-package }} Package"
+    runs-on: ${{ matrix.runs-on }}
     continue-on-error: ${{ matrix.experimental }}
     strategy:
       fail-fast: false
       matrix:
         include:
           # Ubuntu packages.
-          - os: ubuntu-20.04
-            build_family: linux
-            build_package: main-dist-linux
+          - runs-on: [managed-releaser, os-family=Linux, runner-group=releaser]
+            build-family: linux
+            build-package: main-dist-linux
```

Comment:
Hey @powderluv , are you using this workflow downstream? I think setting these jobs to run on self-hosted runners could interfere with that.

Reply:
Ping, @powderluv ? I'm not sure if GitHub Actions has a way to make this configurable for downstream repositories. You might want to fork this workflow if you depend on it.